Userland/Libraries/LibJS/Runtime/ArrayConstructor.cpp

```diff
@@ -95,30 +95,61 @@ JS_DEFINE_NATIVE_FUNCTION(ArrayConstructor::from)
 
     auto* array = Array::create(global_object);
 
+    Function* map_fn = nullptr;
+    if (vm.argument_count() >= 2 && !vm.argument(1).is_undefined()) {
```

Comment:
Just this is fine :)

```suggestion
    if (!vm.argument(1).is_undefined()) {
```

Reply:
oh, that makes sense. Fixed